packages/features/bookings/lib/handleNewBooking/createBooking.ts

```diff
@@ -206,15 +255,17 @@ function buildNewBookingData(params: CreateBookingParams): Prisma.BookingCreateI
           }
         : undefined,
 
-    routedFromRoutingFormReponse: routedFromRoutingFormResponseId
-      ? { connect: { id: routedFromRoutingFormResponseId } }
+    routedFromRoutingFormReponse: routingFormResponseId
+      ? { connect: { id: routingFormResponseId } }
       : undefined,
   };
 
   if (reqBody.recurringEventId) {
     newBookingData.recurringEventId = reqBody.recurringEventId;
   }
 
+  let originalBookingUpdataDataForCancellation: Prisma.BookingUpdateArgs | undefined = undefined;
```

Comment:
NIT: typo 

Should be `originalBookingUpdateDataForCancellation`

Reply:
Ah, took me some time to see the difference 😅